codes/rust/include/tree_node.rs

```diff
@@ -9,6 +9,7 @@ use std::collections::VecDeque;
 use std::rc::Rc;
 
 #[allow(dead_code)]
+#[derive(Debug)]
 pub struct TreeNode {
     pub val: i32,
     pub high: i32,
```

Comment:
height

Reply:
The `high` field is not defined by me, is it ok to be modified?

Reply:
Yes, every file can be modified, especially found something incorrect. All the changes will be reviewed in PR.

Reply:
Actually, welcome to pay more attention to the public module or others' code.